pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -71,6 +71,20 @@ type Plan struct {
 	Phases []Phase `json:"phases"`
 }
 
+// ParameterValueType specifies the type of a parameter value.
+type ParameterValueType string
+
+const (
+	// StringValueType is used for parameter values that are provided as a string.
+	StringValueType ParameterValueType = "string"
```

Comment:
it is also odd that this type is never used... I realize that it is the default... but it seems less readable... more important.. if someone is looking for where in the code it is used... they will find none.   I think I would rather see a case like

```go
		case kudov1beta1.StringValueType:
			fallthrough
		default:
```

Reply:
You're right, this should be used.